frontends/mrxl/mrxl/parse.py

```diff
@@ -18,7 +18,7 @@
 varexpr: CNAME
 ?baseexpr: litexpr | varexpr
 binexpr: baseexpr binop baseexpr
-binop: "+" -> add | "*" -> mul
+binop: "+" -> add | "*" -> mul | "-" -> sub | "/" -> div
```

Comment:
Perhaps the MrXL interpreter should support these too, if we're going to provide parser support for them?

Reply:
I just went to add it, and get this: it already supports them!

Reply:
Amazing!!!!!!!!